README_zh-hans.md

```diff
@@ -173,7 +173,7 @@ checkpoint: 检查点
     - 对所有模型统一的API
 
 1. 更低计算开销，更少的碳排放：
-    - 研究人员可以分享亿训练的模型而非次次从头开始训练
+    - 研究人员可以分享预训练的模型而非次次从头开始训练
```

Comment:
The README file in English is
```bash
Researchers can share trained models instead of always retraining.
```
So I would probably have a translation like
```bash
研究人员可以分享已(经)训练的模型而非每次从头开始训练
```

@JingyaHuang Could you have a look too? You speak better chinese than me 😆 I guess